cdap-ui/app/cdap/components/AppDetailedView/index.js

```diff
@@ -208,13 +202,15 @@ export default class AppDetailedView extends Component {
           />
           <PlusButton mode={PlusButton.MODE.resourcecenter} />
         </div>
-        <OverviewHeader successMessage={this.state.successMessage} />
-        <OverviewMetaSection
-          entity={this.state.entityDetail}
-          onFastActionSuccess={this.goToHome.bind(this)}
-          onFastActionUpdate={this.goToHome.bind(this)}
-          showFullCreationTime={true}
-        />
+        <div>
+          <OverviewHeader successMessage={this.state.successMessage} />
+          <OverviewMetaSection
+            entity={this.state.entityDetail}
+            onFastActionSuccess={this.goToHome.bind(this)}
+            onFastActionUpdate={this.goToHome.bind(this)}
+            showFullCreationTime={true}
+          />
+        </div>
```

Comment:
Do we have to do the same thing for Dataset/Stream Detailed view?

Reply:
Its already taken care of. The detailed views share styling (`app-detailed-view` 🙄) so that should automatically fix it.

Reply:
i mean in DatasetDetailedView, the `OverviewMetaSection` is not inside a <div> container like this. That won't be a problem?

Reply:
The dataset and stream detailed views don't have messages `OverviewHeader and OverviewMetaSection` they only have the meta section. Its only in app detailed view we show the updated preference success message.